acceptancetests/assess_cloud_display.py

```diff
@@ -58,12 +60,13 @@ def remove_display_attributes(cloud):
 def get_clouds(client):
     cloud_list = yaml.safe_load(client.get_juju_output(
         'clouds', '--format', 'yaml', '--local', include_e=False))
-    for cloud_name, cloud in cloud_list.items():
-        if cloud['defined'] == 'built-in':
-            del cloud_list[cloud_name]
-            continue
+    cloud_list_without_builtins = {
```

Comment:
This works different in python3. See comment on L47 about using `list` to get a copy of the keys when mutating the dict while it is being iterated.